src/lib/output/plugins/MarkedPlugin.ts

```diff
@@ -219,6 +215,34 @@ export class MarkedPlugin extends ContextAwareRendererComponent {
         }
     }
 
+    /**
+     * Creates an object with options that are passed to the markdown parser.
+     *
+     * @returns The options object for the markdown parser.
+     */
+    private createMarkedOptions(): Marked.MarkedOptions {
+        const markedOptions = (this.application.options.getValue(
+            "markedOptions"
+        ) ?? {}) as Marked.MarkedOptions;
+
+        if (
+            typeof markedOptions === "object" &&
```

Comment:
We also need a check here for `null`

Reply:
I think you missed the null coalescing a few lines above.

Reply:
Oops! Looks like you're right, should be able to merge tonight.